web3j/src/main/resources/config/application.yml

```diff
@@ -22,10 +22,12 @@ web3j:
   queued-polling: false # set true for queued polling of transaction receipt, will only work with OPEN model
 
 token:
+  contract-address:  # ERC20 contract address (Note: without 0x and in case of empty it will deploy a new contract)
   name: SecureToken
   symbol: STK
   initial-supply: 2 # total tokens minted (value is internally multiplied by 10^18)
   recipient: 45412868cB693e231DfD4ABCCA29D7001501d06f # recipient account
```

Comment:
As discussed, let the user configure an array of recipients (min 1) to whom the sender can transfer tokens equally.